### DISCUSSION TITLE
How do I create a camera-specific yaml file?

### DISCUSSION THREAD

**Discussion by takanotume24** [stella-cv/stella_vslam#236] [Q&A] [answered]
Hello. I want to run OpenVSLAM on my camera.
For example, if I wanted to run OpenVSLAM on a picture taken with the out camera of my LG smartphone (LG Style), what steps should I take to create a yaml file? Such as https://github.com/OpenVSLAM-Community/openvslam/blob/main/example/aist/equirectangular.yaml
I looked at the documentation, but I couldn't find how to make a yaml file, so I asked the question. I'm sorry if this is a common question.

**Comment by ymd-stella** [comment, marked as answer]
See https://github.com/OpenVSLAM-Community/openvslam/blob/f42bfcfabbaf3e4fd0021ac3e283bb89a4312db9/example/tum_rgbd/TUM_RGBD_mono_1.yaml.

Calibrate the camera and update the following parameters.
https://github.com/OpenVSLAM-Community/openvslam/blob/f42bfcfabbaf3e4fd0021ac3e283bb89a4312db9/example/tum_rgbd/TUM_RGBD_mono_1.yaml#L12-L25

**Comment by vromaniello** [comment]
Is it possible to use the same config file also on iOS devices (Iphone XS)? 
Thank you
